src/Comps/style/book.css

```diff
@@ -0,0 +1,149 @@
+.book-div {
```

Comment:
### Your Buttons
![your-buttons](https://user-images.githubusercontent.com/93039359/191901168-e94dce5a-4fe0-4008-ac34-0e923b7dc48c.png)

### Expected Buttons
![expected](https://user-images.githubusercontent.com/93039359/191901201-dc468edf-bebc-42b5-9ff8-4c21735ccd5e.png)

- [ ] Your buttons should be seperated by lines as you can see in the expected image above. Kindly stick to the design to make it professional :+1:

Reply:
This change has been done well